### PR TITLE
[WW-5362] Removes type attribute out of <s:script/> tag

### PR DESCRIPTION
**Pull request by lukaszlenart**
Closes [WW-5362](https://issues.apache.org/jira/browse/WW-5362)

### CODE REVIEW

**Comment by lukaszlenart** [comment]
FYI @sepe81 @gregh3269

**Comment by gregh3269** [comment]
Seems now I get
SEVERE: Exception sending context initialized event to listener instance of class [org.apache.struts2.tiles.StrutsTilesListener]
java.lang.NoClassDefFoundError: org/apache/commons/digester3/Rule
Caused by: java.lang.ClassNotFoundException: org.apache.commons.digester3.Rule
Deps want commons-digester-2.1.jar think this is v3?

**Comment by lukaszlenart** [comment]
What app have you used? This change shouldn't affect Digester

**Comment by gregh3269** [comment]
My app.

**Comment by lukaszlenart** [comment]
This is related to [WW-5347](https://issues.apache.org/jira/browse/WW-5347) - and you are using 6.3.0.1 version as far I see, changes in this PR are unrelated to your problem.

**Comment by gregh3269** [comment]
OK, why would the jars be missing for

.m2/repository/org/apache/struts/struts2-tiles-plugin/6.4.0-SNAPSHOT
.m2/repository/org/apache/struts/struts2-javatemplates-plugin/6.4.0-SNAPSHOT

whilst 
.m2/repository/org/apache/struts/struts2-core/6.4.0-SNAPSHOT are present.

**Comment by lukaszlenart** [comment]
@gregh3269 can we discuss this in the dev@ email list?

**Comment by gregh3269** [comment]
OK, mailing list.

...although I got it working and it seems good.

File: assembly/src/main/resources/template.vm [hunk 1]
```diff
@@ -43,12 +43,12 @@ under the License.
...
     #end
 
-    <script type="text/javascript">
+    <script>
         SyntaxHighlighter.defaults['toolbar'] = false;
         SyntaxHighlighter.all();
     </script>
 #end
-    <script type="text/javascript" language="javascript">
+    <script language="javascript">
```
Comment (sepe81) [inline assembly/src/main/resources/template.vm:51]:
```suggestion
    <script>
```

Should the language attribute also be removed?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#deprecated_attributes states:

>Like the type attribute, this attribute identifies the scripting language in use. Unlike the type attribute, however, this attribute's possible values were never standardized. The type attribute should be used instead.

Comment (lukaszlenart) [reply assembly/src/main/resources/template.vm:51]: Good catch, thanks!

**Comment by lukaszlenart** [comment]
Are we good here?

File: core/src/site/confluence/ww-template-autoexport.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
         color:                 #666;
       }
     </style>
-    <script type="text/javascript" language="javascript">
+    <script language="javascript">
```
Comment (sepe81) [inline core/src/site/confluence/ww-template-autoexport.html:52]:
```suggestion
    <script>
```

one more place

**Review by sepe81 — APPROVED** [review]
LGTM after cleanup for [ww-template-autoexport.html](https://github.com/apache/struts/pull/812/files#diff-ff99f59be56cc691ab5126524d28682baa370e99604b4413584e3cde264483d9)

**Pull request merged**